### ISSUE TITLE
predict-prob always giving 1.00001 as a result for any input

### ISSUE THREAD

**Issue by DiegoZoracKy** [facebookresearch/fastText#925]
Although the results given from a train/test split strategy shows up a high accuracy for the case (~90%), it seems that something is going wrong on predict-prob method as it always answers 1.00001 for any input, even for nonsense ones like "haha", "fasttext", "1234567890".

**Training params:** 
-lr 1.0 -epoch 25 -wordNgrams 2 

**Training data:**
Read 243M words
Number of words:  2273553
Number of labels: 1427

It's worth noting I've trained other models and this is the only case where I could find this weirdness until now.

**Comment by Celebio** [comment]
Hi @DiegoZoracKy ,
Thank you for the feedback.

What do you get when you copy/paste some text from your test set to predict-prob? 

Regards,
Onur

**Comment by DiegoZoracKy** [comment]
Same value. It just gives 1.00001 for any input.

**Comment by secain** [comment]
Do you have any idea now? I meet the same problem

**Comment by pannixilin1** [comment]
same for me
